urdf_parser_plugin/CMakeLists.txt

```diff
@@ -1,13 +1,20 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.12.2)
```

Comment:
Do we really need cmake as big as this version?  This will prevent this from being used on e.g. Bionic, which only has CMake 3.10.  If we don't need a particular feature, I'd rather go with what the rest of this package uses (which I believe is 3.5).

Reply:
CMake 3.5  in 7250733

I probably meant to do `3.10.2` which is the version of CMake in Bionic, but typo'd the 10 as a 12.